### PR TITLE
Extra registeration fields added

### PR DESCRIPTION
**Pull request by rehmansheikh222**

### CODE REVIEW

**Comment by Danyal-Faheem** [comment]
Your commit messages should be of the form, `feat`, `fix`, etc as specificied in [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

**Comment by rehmansheikh222** [comment]
> Your commit messages should be of the form, `feat`, `fix`, etc as specificied in [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

I will keep that in mind in the future

File: tutormyplugin/patches/openedx-lms-development-settings [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+CORS_ORIGIN_WHITELIST.append(
+    "http://apps.local.edly.io")
+CSRF_TRUSTED_ORIGINS.append("http://apps.local.edly.io")
+ALLOWED_HOSTS.append("http://apps.local.edly.io")
+
+ENABLE_DYNAMIC_REGISTRATION_FIELDS = "true"
+MFE_CONFIG["ENABLE_DYNAMIC_REGISTRATION_FIELDS"] = "true"
+REGISTRATION_EXTRA_FIELDS["country"] = "required"
+REGISTRATION_EXTRA_FIELDS["terms_of_service"] = "required"
+MFE_CONFIG["TOS_LINK"] = "https://edly.io"
```
Comment (Danyal-Faheem) [inline tutormyplugin/patches/openedx-lms-development-settings:25]: This works here but there is another patch called `mfe-lms-development-settings` which does the same thing but in terms of making things separate, it is good practice to add MFE_CONFIG changes in there

Reference: https://github.com/overhangio/tutor-mfe?tab=readme-ov-file#customising-mfes

**Review by Danyal-Faheem — APPROVED** [review]
LGTM! 👍🏿

Good work

File: tutormyplugin/patches/openedx-lms-development-settings (hunk 1 above)
Comment (rehmansheikh222) [reply tutormyplugin/patches/openedx-lms-development-settings:25]: Sure, I will make the suggested changes

**Pull request merged**